hw20/js/28students.js

```diff
@@ -0,0 +1,221 @@
+class Human {
+    name = null;
+    surname = null;
+    age = null;
+
+    constructor(person = { name: 'John', surname: 'Doe', age: 30 }) {
+        this.name = person.name;
+        this.surname = person.surname;
+        this.age = person.age;
+    }
+
+    getFullName() {
+        return `${this.name} ${this.surname}`;
+    }
+    setFullName(fullName) {
+        const [ name, surname ] = fullName.split(' ');
+
+        this.name = name;
+        this.surname = surname;
+    }
+}
+
+class Student extends Human {
+    marks = [];
+
+    constructor(person = { name: 'JohnS', surname: 'DoeS', age: 18, marks: [] }) {
+        super(person);
+
+        this.marks = person.marks;
+    }
+
+    getMarks() {
+        return this.marks;
+    }
+    getAverageMark() {
+        return this.marks.reduce((total, mark) => total + mark, 0) / this.marks.length;
+    }
```

Comment:
please add a blank line between methods. Fix it everywhere

Reply:
```+```